docs/integrations/engines.md

```diff
@@ -226,12 +226,12 @@ pip install "sqlmesh[redshift]"
 
 The operator requires an [Airflow connection](https://airflow.apache.org/docs/apache-airflow/stable/howto/connection.html) to determine the target Redshift account. Refer to [AmazonRedshiftConnection](https://airflow.apache.org/docs/apache-airflow-providers-amazon/stable/connections/redshift.html#authenticating-to-amazon-redshift) for details on how to define a connection string.
 
-By default, the connection ID is set to `sqlmesh_redshift_default`, but it can be overridden using the `default_engine_operator_args` parameter to the `SQLMeshAirflow` instance as in the example below:
+By default, the connection ID is set to `sqlmesh_redshift_default`, but it can be overridden using the `engine_operator_args` parameter to the `SQLMeshAirflow` instance as in the example below:
 ```python linenums="1"
 sqlmesh_airflow = SQLMeshAirflow(
     "redshift",
-    default_engine_operator_args={
-        "sqlmesh_redshift_conn_id": "<Connection ID>"
+    engine_operator_args={
+        "conn_id": "<Connection ID>"
```

Comment:
I figured this is the correct name from the following code in operators/redshift.py:

```python
class SQLMeshRedshiftOperator(BaseSQLOperator):
    """The operator that evaluates a SQLMesh model snapshot on Redshift cluster

    Args:
        target: The target that will be executed by this operator instance.
        conn_id: The Airflow connection id for the Redshift target.
    """

    def __init__(
        self,
        target: BaseTarget,
        **kwargs: t.Any,
    ) -> None:
        super().__init__(conn_id=SQLMeshRedshiftHook.default_conn_name, **kwargs)
```